cmd/thermal-recorder/config_test.go

```diff
@@ -34,13 +34,17 @@ import (
 	"github.com/TheCacophonyProject/window"
 )
 
+var locationFileName = "../../_release/location.yaml"
```

Comment:
If you need one for testing don't put it in the _release directory
That directory is for stuff to be included the package for deployment.
Make a `test-data`  directory or something

Reply:
Oh ok.  I did think it was for testing lol.  Will do.